android/app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java

```diff
@@ -77,10 +77,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         //set up view model
         locationsFragmentViewModel = ViewModelProviders.of(this).get(LocationsFragmentViewModel.class);
         searchText = locationsFragmentViewModel.getSearchText();
-        locationsFragmentViewModel.getLocations().observe(LocationsFragment.this, microlocations ->  {
+        locationsFragmentViewModel.getLocations().observe(LocationsFragment.this, microlocations -> {
                 locations.clear();
```

Comment:
I don't understand why you had to edit ```LocationsFragment``` if you are creating ViewModel for ```SearchActivity```. Was changing this file absolutely necessary ?

Reply:
This was a silly error 😅